example/e2e/helpers.ts

```diff
@@ -437,3 +404,32 @@ export async function disableSync() {
     await device.disableSynchronization();
   }
 }
+
+export async function setupRavelinConfigWithURL(config: ravelinConfig) {
+  const recommendationURL = process.env.RAVELIN_REC_URL;
+  const ravelinConfig = processJSONFile('./configs/ravelin.json', {
+    apiConfiguration: {
+      judoId: process.env.JUDO_ID,
+    },
+    authorization: {
+      token: process.env.API_TEST_TOKEN,
+      secret: process.env.API_TEST_SECRET,
```

Comment:
Don't we require Payment Session to be used with Recommendation feature? Is token and secret okay here? Sorry if that's stupid question, but I noticed we have error message "To use the recommendation feature, it is mandatory to have a payment session.", and therefore asking.